.gitignore

```diff
@@ -1,4 +1,5 @@
 neurobot.db
+neurobot-test.db
```

Comment:
It would be of interest to you to check out, how I setup test dbs in `db_test.go` with a setUp() and tearDown() function. There  are SQLs I execute to setup the empty database to a state I want to test against. Might not be the best setup, but so far, it grew organically for me.